scripts/manpage_generation.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Directory to store the generated manpages
+MANPAGE_DIR="../man"
+
+# Generate the manpage using argparse-manpage
+PYTHONPATH=. argparse-manpage --pyfile man/__init__.py --function get_parser --manual-title="General Commands Manual" --description="Automates the conversion of Red Hat Enterprise Linux derivative distributions to Red Hat Enterprise Linux." --project-name "Convert2RHEL" --prog="convert2rhel" > "$MANPAGE_DIR/convert2rhel.8"
```

Comment:
```suggestion
# Generate a file with convert2rhel synopsis for argparse-manpage
python -c 'from convert2rhel import toolopts; print("[synopsis]\n."+toolopts.CLI.usage())' > man/synopsis
# Generate the manpage using argparse-manpage
PYTHONPATH=. argparse-manpage --pyfile man/__init__.py --function get_parser --manual-title="General Commands Manual" --description="Automates the conversion of Red Hat Enterprise Linux derivative distributions to Red Hat Enterprise Linux." --project-name "Convert2RHEL" --prog="convert2rhel" --include man/synopsis > "$MANPAGE_DIR/convert2rhel.8"
```

If https://github.com/oamg/convert2rhel/pull/917/commits/15535033e1fb6d25c0a38a6e6bd81cf8ae03b0b0 is accepted/merged.

Reply:
You are missing the man/distribution part in this one: 

```suggestion
# Generate the manpage using argparse-manpage
PYTHONPATH=. argparse-manpage --pyfile man/__init__.py --function get_parser --manual-title="General Commands Manual" --description="Automates the conversion of Red Hat Enterprise Linux derivative distributions to Red Hat Enterprise Linux." --project-name "Convert2RHEL" --prog="convert2rhel" --include man/distribution --include man/synopsis > "$MANPAGE_DIR/convert2rhel.8"
```

Reply:
I fixed this!